### ISSUE TITLE
Default Branch Has a Failing Build

### ISSUE THREAD

**Issue by RobbieMcKinstry** [pulumi/pulumi-winget#16]
### What happened?

The main branch is currently failing. (https://github.com/pulumi/pulumi-winget/actions/runs/3291411698/jobs/5425526918)

I'm unsure if this is a blocker for the release today. Just wanted to raise this issue to track the work in case it is.

### Steps to reproduce

Observe https://github.com/pulumi/pulumi-winget/actions/runs/3291411698/jobs/5425526918

### Expected Behavior

Green build

### Actual Behavior

Red build

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

**Comment by Zaid-Ajaj** [comment]
This is a false positive, Robbie. In the step `Generate Pulumi MSI`, we skipped generating the installer and download URL because there was no need for it
```
Version v3.43.1 of Pulumi MSI is already published, skipping...
```
Then the `Push WinGet Manifest` fails because there was no download-url.txt file present which is correct since we didn't create any build artifacts.

I will change the workflow so that it doesn't fail when there is nothing to do but this is definitely non-p0/p1

**Comment by Zaid-Ajaj** [comment]
addressed by #17

**Comment by RobbieMcKinstry** [comment]
Thanks very much! ❤️  I didn't know, so I wanted to get some eyes from someone more knowledgeable, just in case! You're the best, Zaid!
